python/enthic/compute_stats.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+"""
+=========================================================================
+Compute statistics from companies's data after complete and check it
+=========================================================================
+
+PROGRAM BY PAPIT SASU, 2019
+
+Coding Rules:
+
+- Snake case for variables.
+- Only argument is configuration file.
+- No output or print, just log and files.
+"""
+import json
+import math
+
+def convert_data_to_tree(rawData):
+    """
+    The def convert_data_to_tree takes an empty CompteDeResultat tree
+    from "arbre_compte_de_resultat.json" file, and fill it with data from enthic API
```

Comment:
Please add a properly restructured text docsrting documenting the code. You can use https://livesphinx.herokuapp.com/ to help formatting.